molecule/delegated/tests/util/util.py

```diff
@@ -89,6 +89,19 @@ def get_from_url(url, binary=False):
     return content
 
 
+def get_centos_repo_key(host, summary):
```

Comment:
I would argue it would be good to add Robins UserAgent spoof from collection-services

Reply:
Refactored `get_from_url` function in `util.py`:
```
def get_from_url(url, binary=False):
    # Create a request object with a faked User-Agent header.
    # Some websites like https://pkg.osquery.io/rpm/GPG need this, otherwise they will return http 403 forbidden.
    req = urllib.request.Request(
        url, headers={"User-Agent": "Mozilla/5.0 (Linux x86_64) Chrome/103.0.0.0"}
    )
    # Open the URL with the custom request object
    resource = urllib.request.urlopen(req)

    if not binary:
        encoding = resource.headers.get_content_charset()
        if encoding is None:
            encoding = "utf-8"
        content = resource.read().decode(encoding)
    else:
        content = resource.read()

    return content
```